docs/machine-learning/how-to-guides/getting-started-dataframe.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Getting started with DataFrames
+description: Learn how to use DataFrame to manipulate and prepare data.
+author: beccam
+ms.author: beccam
+ms.date: 1/13/2023
+ms.custom: mvc, how-to, title-hack-0625
+ms.topic: how-to
+---
+
+# What is the DataFrame?
+
+Learn how to get started with DataFrames. [DataFrames](https://learn.microsoft.com/dotnet/api/microsoft.data.analysis.dataframe) are a two-dimensional data structure for storing and manipulating data. DataFrames help with preparation of data for a machine learning model. DataFrames can also be used for data manipulation unrelated to machine learning. 
```

Comment:
Question for multiple places - are these doc links supposed to be the url or some other format?

Reply:
They should be xrefs but URL works for now.